frontend/src/components/worksheets/items/WorksheetItem.js

```diff
@@ -140,7 +141,7 @@ class TableWorksheetRow extends React.Component {
         var item = this.props.item;
         var className = /*'type-worksheet' + */ this.props.focused ? ' focused' : '';
         return (
-            <tr className={className}>
+            <tr className={className} id={this.props.id}>
```

Comment:
can pull out id with other props since you do it already

Reply:
Not in the `render()` function, though. will just leave it as is